README.md

```diff
@@ -219,12 +232,12 @@ lançant la commande suivante :
 bundle exec ruby bin/reload_mock_backend.rb
 ```
 
-## Contribution
+## <a name="ajouter-donnees-de-test"></a> Contribution : Ajouter des données de test 
 
-Si vous êtes développeur, référez-vous à la section [Développement](#developpement) ci-dessous.
+**Si vous êtes développeur**, référez-vous à la section [Développement](#developpement) ci-dessous.
 
-Si vous êtes un fournisseur de données et que vous voulez ajouter des données
-d'une API non existante encore, veuillez vous réferer au dossier
+**Si vous êtes un fournisseur de données** et que vous voulez ajouter des données
```

Comment:
@skelz0r je ne comprends pas du tout ce paragraphe à destination des FD, notamment la phrase " si ce n'est pas le cas"

Reply:
Si il s'agit d'une API existante.

Reply:
Tu veux dire une API qui a déjà un dossier dans /payload ?

Reply:
Oui. imo y'a du avoir un micmac sur une itération mais en gros faudrait mettre "Si vous voulez intégrer des données sur une API existante"